### ISSUE TITLE
Beginner Corporations should be selectable with expansions

### ISSUE THREAD

**Issue by peonmodel** [terraforming-mars/terraforming-mars#6873]
Right now, the _only_ way to play with beginner corporation is playing completely without any expansions (excluding corporate era), players are forced to play a gimped game to even try it

THEN players are _forced_ to play beginner corporations if they select beginner

this is quite hostile to new players, prelude expansion helps to speed up the game (honestly TM is kinda long and new players might be resistant to spend that much time)

when introducing the game to new players, i often give them the option to play with a normal corp or a beginner corp, since beginner corp can be boring but really worth it if the starting hand is awesome

it is not even selectable in the custom corporation list

it should be an option akin to the merger prelude, unlike other corporations, its possible to multiple players to play the beginner corp at the same time, just like how multiple copies of merger prelude is possible with the merger option

**Comment by Borbarad2** [comment]
I have no opionion or comment at that issue, but I want to add that if you are playing merger option, you are forced to play with prelude cards. But if I want to play the base game (or Venus or Colony only) with merger option, I can't do that, because i am forced to play with prelude.
It would be nice if we would have the merger option without the need of prelude.

**Comment by kberg** [comment]
@peonmodel Hm.

To give a counterpoint: the point of the Beginner Corp is to keep beginners from making important decisions right when the game is starting. Prelude adds important decisions that everyone has to make at the start of the game.

Fryx talks about the role of BC a few times on BGG, which generally supports that perspective.

**Comment by peonmodel** [comment]
> @peonmodel Hm.
> 
> To give a counterpoint: the point of the Beginner Corp is to keep beginners from making important decisions right when the game is starting. Prelude adds important decisions that everyone has to make at the start of the game.
> 
> Fryx talks about the role of BC a few times on BGG, which generally supports that perspective.

yes, but this is inconsistent with how this app allows, one could play with septum tribus without turmoil expansion, if one chooses so, with only a warning pop up, using the custom corporation list

unlike beginner corp which is not even inside the custom corp list, _despite_ being an officially designed corp & already coded in the app

unlike the obviously gimped game playing septum tribus without turmoil expansion,  a beginner corp _can_ actually be the optimal choice for an experienced player if the starting 10 cards is awesome, situationally just like every other corps

**Comment by kberg** [comment]
I've thought about this some more, and I don't think it's worth implementing. This is a fine thing for someone that wants to fork the repo can implement, so i'm going to close this. Feel free to add comments, I'll receive them.
